target/linux/omap/image/Makefile

```diff
@@ -44,29 +44,33 @@ endef
 #uboot-omap-am335x_evm uboot-omap-omap3_beagle uboot-omap-omap3_overo uboot-omap-omap4_panda
 
 define Device/ti_am335x-evm
-  DEVICE_TITLE := TI AM335x EVM
+  DEVICE_VENDOR := Texas Instruments
+  DEVICE_MODEL := TI AM335x EVM
   DEVICE_DTS := am335x-evm
 endef
 
 TARGET_DEVICES += ti_am335x-evm
 
 define Device/ti_am335x-bone-black
-  DEVICE_TITLE := TI AM335x BeagleBone Black
+  DEVICE_VENDOR := None
+  DEVICE_MODEL := TI AM335x BeagleBone Black
   DEVICE_DTS := am335x-boneblack
 endef
 
 TARGET_DEVICES += ti_am335x-bone-black
 
 define Device/ti_omap4-panda
-  DEVICE_TITLE := OMAP4 TI pandaboard
+  DEVICE_VENDOR := None
+  DEVICE_MODEL := OMAP4 TI pandaboard
```

Comment:
I'm not sure whether Texas instruments really is the vendor here or whether this device is just targetting the TI architecture ...

Reply:
I think the vendor is PandaBoard.org.

Reply:
I've added PandaBoard.org as vendor.